src/server.js

```diff
@@ -470,3 +466,13 @@ function auth(data) {
 		localAuth(client, data.user, data.password, authCallback);
 	}
 }
+
+function reverseDnsLookup(ip, callback) {
```

Comment:
Rewrote this function to be an independent helper. This improves authentication flow (e.g. remember me in webirc most likely did not work before).